common/djangoapps/util/milestones_helpers.py

```diff
@@ -313,6 +315,16 @@ def remove_content_references(content_id):
     return milestones_api.remove_content_references(content_id)
 
 
+def any_unfulfilled_milestones(course_id, user_id):
+    """ Returns a boolean if user has any unfulfilled milestones """
+    try:
+        return bool(
+            get_course_milestones_fulfillment_paths(course_id, {"id": user_id})
+        )
+    except InvalidMilestoneRelationshipTypeException:
```

Comment:
Okay -- there are other exceptions which may be raised by the milestones api as well, so you might look into those, too